Decred Wallet/Features/More/More Features/Debug/Debug.storyboard

```diff
@@ -121,7 +142,129 @@
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="GAo-5d-y9o" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="362" y="1596"/>
+            <point key="canvasLocation" x="-788" y="1742"/>
+        </scene>
+        <!--Connected Peer View Controller-->
+        <scene sceneID="GBY-sX-XPL">
+            <objects>
+                <viewController id="E2C-vA-WN7" customClass="ConnectedPeerViewController" customModule="Decred_Wallet" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="f5f-Ix-Gtx">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
```

Comment:
The ban score of the last item is chopped off the bottom

<img width="361" alt="Screenshot 2021-02-21 at 02 38 31" src="https://user-images.githubusercontent.com/794430/108613060-7d81c780-73ee-11eb-9505-1dd0d4f45359.png">

the nav bar should tally with the rest, there shouldn't be a back text and the connected peer text color should be changed.
<img width="352" alt="Screenshot 2021-02-21 at 02 37 02" src="https://user-images.githubusercontent.com/794430/108613090-dcdfd780-73ee-11eb-9c03-6ff6bfe618ac.png">